src/components/ChatLog.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+
+const ChatLog = (props) => {
+    const chatEntryComponents = props.entries.map((entry) => {
+        return (
+        <ChatEntry
+            id={entry.id}
+            sender={entry.sender}
+            body={entry.body}
+            timeStamp={entry.timeStamp}
+            isLiked={entry.isLiked}
+            onUpdate={props.onUpdateChatMessages}></ChatEntry>
+        );
+    });
+    return (
+        <section class="chat-log">{chatEntryComponents}</section>
```

Comment:
```suggestion
        <section className="chat-log">{chatEntryComponents}</section>
```
jsx using className unlike class in html